netlib/websockets/frame.py

```diff
@@ -128,17 +141,16 @@ def __bytes__(self):
             # '!Q' = pack as 64 bit unsigned long long
             # add 8 bytes extended payload length
             b += struct.pack('!Q', self.payload_length)
+        else:
+            raise ValueError("Payload length exceeds 64bit integer")
```

Comment:
Should probably be RuntimeError (or something better), because __bytes__ does not really accept a parameter/value?